CONTRIBUTING.md

```diff
@@ -11,7 +11,7 @@ These are just guidelines, not rules. Use your best judgment, and feel free to p
 
 This project adheres to the Contributor Covenant [code of conduct](CODE_OF_CONDUCT.md).
```

Comment:
Unrelated but : on line 5 above, the aptly link seems invalid.

Reply:
fixed